packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/sign-in-page/react-native/signin.md

```diff
@@ -0,0 +1,32 @@
+The async `signIn` method will automatically redirect users to your Okta organization for authentication. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted.
```

Comment:
removed "will" and added an s to redirect and emit, added "that" in front of your - Make sure that your..

The async `signIn` method automatically redirects users to your Okta organization for authentication. It emits an event once a user successfully signs in. Make sure that your event listeners are mounted and unmounted.